README.md

```diff
@@ -111,59 +111,67 @@ and will not work with objects. If this functionality is needed,
 then Lodash/Underscore is the better option.*
 
 1. [_.each](#_each)
-1. [_.every](#_every)
-1. [_.filter](#_filter)
-1. [_.groupBy](#_groupby)
-1. [_.includes](#_includes)
-1. [_.map](#_map)
-1. [_.minBy and _.maxBy](#_minby-and-_maxby)
-1. [_.orderBy](#_sortby-and-_orderby)
-1. [_.pluck](#_pluck)
-1. [_.range](#_range)
-1. [_.reduce](#_reduce)
-1. [_.reduceRight](#_reduceright)
-1. [_.size](#_size)
-1. [_.some](#_some)
-1. [_.sortBy](#_sortby-and-_orderby)
-1. [_.uniq](#_uniq)
+2. [_.every](#_every)
```

Comment:
if you just write `1.` it will auto create an order list with correct numbers.

Reply:
yeah sorry markdown vscode extension, will disable it